cardano-config/src/Cardano/Config/Orphanage.hs

```diff
@@ -1,14 +1,85 @@
-{-# LANGUAGE StandaloneDeriving #-}
```

Comment:
@Jimbo4350, the `*.Orphanage` modules are supposedly intended for orphan instances, not for just about everything -- this is so they can:

1. have `-Wno-orphans`, and
2. be imported as `import *.Orphanage ()`

I think this is confusing this purpose..

Reply:
@deepfire Agreed. Let me think about where to put some of this stuff.

Reply:
@deepfire Pushed changes.